Header header

float64 starter_consistency

float64 time_stamp
float64 ender_consistency
float64 counter
float64 check_sum

uint16 itc_info
bool hardware_fail
bool sgu_fail
float64 horizontal_misalign
float64 absorb_blind
float64 distort_blind
uint8 message_counter
uint32 message_crc
